Mathlib/Topology/EMetricSpace/Basic.lean

```diff
@@ -89,6 +90,17 @@ attribute [instance] PseudoEMetricSpace.toUniformSpace
 
 /- Pseudoemetric spaces are less common than metric spaces. Therefore, we work in a dedicated
 namespace, while notions associated to metric spaces are mostly in the root namespace. -/
+
+/-- Two pseudo emetric space structures with the same edistance function coincide. -/
+@[ext]
+theorem PseudoEMetricSpace.ext {α : Type*} {m m' : PseudoEMetricSpace α}
```

Comment:
maybe this should be `protected theorem` (to avoid confusion with other lemmas called `ext`)